### ISSUE TITLE
Cursed Talismans

### ISSUE THREAD

**Issue by Pueken** [ccshiro/cc-buglist#2558]
The Quest "Cursed Talismans" in Hellfire Peninsula is bugged, even after killing 10-15 orcs, not a single Talisman dropped, pretty sure the droprate is 0%

**Comment by GogoCC** [comment]
Not true, I completed this quest yesterday by killing Bleeding Hollow Peons. Check wowhead's comments before opening an uissue

**Comment by Bahalol** [comment]
Despite the report's conclusion being incorrect, the concern is not misplaced. According to Wowwiki and Wowhead, other mobs besides Bleeding Hollow Peons should drop Cursed Talismans. Imagine attempting to do this quest with a population of 2000+ players.
